content/kubernetes/re-clusters/multi-namespace.md

```diff
@@ -0,0 +1,176 @@
+linktitle: Manage multiple namespaces
+description: Redis Enterprise for Kubernetes allows you to deploy to multiple namespaces within your Kubernetes cluster. This article shows you how to configure your Redis Enterprise cluster to connect to databases in multiple namespaces
+weight: 17
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/re-clusters/multi-namespace.md,
+    /kubernetes/re-databases/multi-namespace/,
+]
+
+---
+
+A single Redis Enterprise cluster (REC) can manage multiple Redis Enterprise database objects (REDB) in multiple namespaces within the same Kubernetes cluster.
+
+To learn more designing a multi-namespace Redis Enterprise cluster, see [flexible deployment options]({{<relref "/kubernetes/deployment/deployment-options.md">}}).
+
+## Prerequisites
+
+Before configuring a multi-namespace deployment, you must have a running [Redis Enterprise cluster (REC)]({{<relref "/kubernetes/deployment/quick-start.md">}}). See more information in the [deployment]({{<relref "/kubernetes/deployment/">}}) section. 
+
+## Create role and role binding for managed namespaces
+
+Both the operator and the RedisEnterpriseCluster (REC) resource need access to each namespace the REC will manage. For each managed namespace, create a `role.yaml` and `role_binding.yaml` file within the managed namespace, as shown in the examples below.
+
+Replace `<rec-namespace>` with the namespace the REC resides in.
+Replace `<service-account-name>` with the your own value (by default this is the REC name).
+
+`role.yaml` example: 
+
+```yaml
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: redb-role
+  labels:
+    app: redis-enterprise
+rules:
+  - apiGroups:
+      - app.redislabs.com
+    resources: ["redisenterpriseclusters", "redisenterpriseclusters/status", "redisenterpriseclusters/finalizers",
+                "redisenterprisedatabases", "redisenterprisedatabases/status", "redisenterprisedatabases/finalizers"]
+    verbs: ["delete", "deletecollection", "get", "list", "patch", "create", "update", "watch"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["update", "get", "read", "list", "listallnamespaces", "watch", "watchlist",
+            "watchlistallnamespaces", "create","patch","replace","delete","deletecollection"]
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["create"]
+  - apiGroups: [""]
+    resources: ["services"]
+    verbs: ["get", "watch", "list", "update", "patch", "create", "delete"]
+```
+
+`role_binding.yaml` example:
+
+```yaml
+ kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: redb-role
+  labels:
+    app: redis-enterprise
+subjects:
+- kind: ServiceAccount
+  name: redis-enterprise-operator
+  namespace: <rec-namespace>
+- kind: ServiceAccount
+  name: redis-enterprise-admission
+  namespace: <rec-namespace>
```

Comment:
We don't have such a ServiceAccount (I think it's a remnant of old days).

I can see this is also the case with the manifests we provide in k8s-docs/multi-namespace-redb. Let me verify if we can remove it altogether.

Reply:
Removed from `multi-namespace-redb/role_binding.yaml` - will be synced to [RedisLabs/redis-enterprise-k8s-docs](RedisLabs/redis-enterprise-k8s-docs) upon next release.

Reply:
I've added DOC-1827 to make sure this is updated before release.